docs/accessibility/core-concepts/accessibility-score.mdx

```diff
@@ -36,6 +36,8 @@ The accessibility score calculation applies the following weights:
 - Calculate the total weight of failed checks.
 - Compute the score as: `Score = (Passed Weights) / (Passed Weights + Failed Weights)`
 
+**Note**: Incomplete and Ignored element statuses do not have any effect on your score. Only clear failures of a rule are counted.
```

Comment:
This is unrelated to the PR, but clears something up for an existing Cypress Accessibility user I talked to.